cluster/cluster.yaml

```diff
@@ -1473,6 +1473,14 @@ Resources:
                 - ''
                 - - 'arn:aws:iam::{{.Cluster.InfrastructureAccount | getAWSAccountID}}:role/'
                   - !Ref MasterIAMRole
+          - Action:
+              - 'sts:AssumeRoleWithWebIdentity'
+            Effect: Allow
+            Principal:
+              Federated: "arn:aws:iam::{{ .Cluster.InfrastructureAccount | getAWSAccountID }}:oidc-provider/{{.Cluster.LocalID}}.{{.Values.hosted_zone}}"
+            Condition:
+              StringLike:
```

Comment:
Any particular reason for `StringLike`? It is `StringEquals` elsewhere

Reply:
No, looks like a mistake. Thanks for fixing it.